conda/recipes/rmm/meta.yaml

```diff
@@ -40,22 +41,36 @@ requirements:
     - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - cuda-version {{ cuda_version }}
```

Comment:
Do we need cuda-version here? We don't have it for any of the other compiler `build` sections (in any RAPIDS package) AFAIK.

Reply:
Yes, we will be using `cuda-version` to select the CUDA 12 compiler version that we use. Resolving this.